doc/source/general/skid.rst

```diff
@@ -0,0 +1,506 @@
+    at the end of the note when the sustained duration of the note
+    is over.
+    This time setting is not modified by :ref:`tempo <skid_tempo>`,
+    because it represents the instrument's physical properties that
+    don't change when the song goes faster.
+
+Default settings for the ADSR envelopes for all voices in the
+SKID chip are:
+
+* Attack = 0.0s
+* Decay = 0.0s
+* Sustain = 1.0
+* Release = 0.0s
+
+This produces a sound that will suddenly start and suddenly stop in a very
+artificial sounding way.
+
+It is possible to make the chip describe different shaped envelopes by
+using degenerate values for some of these settings.  For some examples:
+
+Below are settings you might use for a "staccato" type of instrument,
+such as a drum, that is incapable of holding a sustained note and
+instead just fades right away whenever you hit a note:
+
+* Attack = 0.0s
+* Decay = 0.2s   (Note decays immediatly on being struck)
+* Sustain = 0.0  (and it decays to zero, so you can't "hold" the note).
+* Release = 0.0s (Because Sustain is zero, this setting doesn't matter).
+
+Below are settings you might use for an instrument with a strong
+"wow wow" effect, where it takes time to reach full volume and you
+have to hold a note in for a half second or so before you really hear it:
+
+* Attack = 0.5s (takes a whole half second to reach full volume)
+* Decay = 0s (stays at full volume once it's there)
+* Sustain = 1.0 (stays at full volume once it's there)
+* Release = 0.2 (and fades fairly fast when released, but not *immediately*).
+
+.. _skid_letter_frequency:
+
+Letter Frequency Notation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The SKID chip contains an interior lookup system that makes it possible,
+anywhere the chip expects you to give a frequency in Hertz, to instead
+specify its frequency using the more familiar
+`"letter notes". <https://en.wikipedia.org/wiki/Scientific_pitch_notation>`_
+
+To do this, you use a string in the following format:
+
+1. Mandatory: First character is the note letter, one of 
+   "C","D","E","F","G","A", "B", or "R"(to mean "rest").
+2. Optonal: Followed by an optional character, "#" or "b" for "sharp" or
+   "flat".  Note the ASCII characters hash ("#") and lower-case "B" ('b')
+   are used for "sharp" and "flat" in place of the Unicode characters
+   U+266F and U+266D (which are the more proper "sharp" and "flat"
+   characters, but they are cumbersome to type on most keyboards.)
+3. Mandatory: The last character is a digit indicating which octave
+   number (0 through 7) this note is in.  (4 is the "middle" octave
+   that starts with "middle C" on a piano keyboard.)
+
+Examples:  ``"C4"`` is middle C.  ``"C#4"`` is the C-sharp just one half
+step above middle C.  ``"Db4"`` is the D-flat that is in fact the same
+thing as ``"C#4"``.  ``"B3"`` is the B that is just to the left of
+middle C (Note the octave numbering starts with C at the bottom of the
+octave, not A, thus why A3 is adjacent to C4.)
```

Comment:
Dangit, I proofread this once over but missed this.  Where it says "A3 is adjacent" it should say "B3 is adjacent".